biz-scenario/2 Table.sql

```diff
@@ -0,0 +1,31 @@
+use ArmyDB
+GO
+
+DROP table if EXISTS dbo.Army
+GO
+
+CREATE TABLE dbo.Army(
+    ArmyID int not null identity primary key,
+    FirstName varchar (20) not null,
+        CONSTRAINT c_Army_First_Name_can_not_be_blank CHECK (firstname <> ''),
+    LastName varchar (20) not null 
+        CONSTRAINT c_Army_Last_Name_can_not_be_blank CHECK (LastName <> ''),
+    SSN char (10) not null
+        constraint u_Army_SSN UNIQUE,
+        CONSTRAINT c_Army_SSN_must_have_a_dash CHECK  (ssn like'%-%'),
+    DateOfBirth date not null,
+    PlaceOfResidence varchar (35) not NULL
+        CONSTRAINT c_Army_place_of_residence_can_not_be_blank CHECK (PlaceOfResidence <> ''),
+    DateOfEnlistment date not null,
```

Comment:
-- LL this constraint should go after all the columns as it is a multi column constraint